File: bronze/2738/c++/main.cpp
#include <iostream>

int main()
{
    int x, y;
    std::cin >> y >> x;
    int array[x*y * 2];
    int answer[x*y];

    for (int i = 0; i < (x*y *2); i++) {
        std::cin >> array[i];
    }
    
    for (int i = 0; i < (x*y); i++) {
        answer[i] = array[i] + array[x*y +i];

        if (i > 0 && i % 3 == 0) {
            std::cout << '\n';
        }
        if ((i +1) % 3 == 0) {
            std::cout << answer[i];
        } else {
            std::cout << answer[i] << ' ';
        }
    }
    return 0;
}

File: bronze/2738/python/main.py

if (__name__ == "__main__"):
    n, m = map(int, input().split())
    array = list(); answer = list()
    for i in range(n * 2):
        array.append(list(map(int, input().split())))

    for i in range(n):
        for j in range(m):
            answer.append(array[i][j] + array[i + n][j])
    for i in range(n*m):
        if ((i +1) % m == 0):
            print(answer[i])
        else :
            print(answer[i], end=' ')
